app/javascript/src/dashboard.js

```diff
@@ -68,15 +68,18 @@ $('document').ready(() => {
         }
       })
 
+      const possibleCaseNumberPrefixes = ['CINA', 'TPR']
```

Comment:
This is probably going to break with our new CASAs with new prefixes. It's fine for now as long as it doesn't break no-prefix case numbers